src/bootstrap.coffee

```diff
@@ -16,7 +16,7 @@ semverRegex = require('semver-regex')
 
 userConfig = {}
 
-DuplicateUuidError = message: '"uuid" must be unique.'
+DuplicateUuidError = (err) -> _.startsWith(err.message, '"uuid" must be unique')
```

Comment:
Why the change? Also it should probably be in a commit by itself if it's unrelated to the api key stuff

Reply:
It's explained in the commit message - the API has had a bug where it added an extra dot at the end of the message, and I want to make the supervisor resilient to that bug. Added it inside this commit since it's a small change that doesn't affect behavior much, but I guess I can move it into a separate commit.
It is still *sort of* related since if this check fails then the supervisor will keep retrying the provisioning instead of doing the key echxange to register the deviceApiKey.

Reply:
Ok, I'd prefer it to be a separate commit, but otherwise I'm good with it, or even with `_.includes`

Reply:
Moved to a separate commit